spec/models/notification_spec.rb

```diff
@@ -19,7 +19,7 @@
     @note = Notification.new(@opts)
   end
 
-  it 'destoys the associated notification_actor' do
+  it 'destroys the associated notification_actor' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.